format/Schema.fbs

```diff
@@ -125,9 +125,9 @@ table FixedSizeList {
 /// may be set in the metadata for this field.
 ///
 /// In a field with Map type, the field has a child Struct field, which then
-/// has two children: key type and the second the value type. The names of the
-/// child fields may be respectively "entries", "key", and "value", but this is
-/// not enforced.
+/// has two children: the first of the key type and second of the value type.
```

Comment:
I'm not good at English but we don't need one more `the`?

```suggestion
/// has two children: the first of the key type and the second of the value type.
```

Reply:
It works grammatically either way, but the extra "the" isn't strictly necessary. Definite articles often need not be repeated in lists.  I, personally, prefer the shorter form but defer to maintainers.

See https://english.stackexchange.com/questions/178767/definite-article-repetition-in-lists for a spot of context.

Reply:
OK. I'll merge this PR in the next week if nobody objects this PR.